src/main/java/net/aufdemrand/denizen/scripts/commands/BukkitCommandRegistry.java

```diff
@@ -2139,6 +2139,10 @@ public void registerCoreMembers() {
         // Use to edit the NBT of a player's item in hand.
         // - nbt <player.item_in_hand> "MyCustomNBT.Owner:<player>" "save:edited"
         // - inventory set "slot:<player.item_in_hand.slot>" "o:<entry[edited].new_item>"
+        // @Usage
+        // Use to remove an NBT tag from a player's item in hand.
+        // - nbt <player.item_in_hand> "MyCustomNBT.Owner:" "save:item"
+        // - inventory set "slot:<player.item_in_hand.slot>" "o:<entry[item].new_item>"

```

Comment:
I actually... don't think I meant to make it that way, I think I meant to go back after I decided whether it should be "!" or something else. @mcmonkey4eva what do you think about this functionality?